docs/external-classes.json

```diff
@@ -1,98 +1,50 @@
 [{
   "name": "Google\\Auth\\",
-  "uri": "https://github.com/google/google-auth-library-php/blob/master/src/%s.php"
+  "uri": "https://github.com/google/google-auth-library-php/blob/master/src/%s.html"
 }, {
   "name": "Google\\GAX\\",
-  "uri": "https://github.com/googleapis/gax-php/tree/master/src/%s.php"
+  "uri": "https://googleapis.github.io/gax-php/master/Google/GAX/%s.html"
```

Comment:
I am happy to do that, updated.

My only concern is that we will need to keep this updated as new docs are published. I am happy to try pointing at the most recent docs, and if we find problems keeping everything updated we can try a different approach.